gms/feature_service.cc

```diff
@@ -71,6 +71,9 @@ feature_config feature_config_from_db_config(const db::config& cfg, std::set<sst
     if (!cfg.check_experimental(db::experimental_features_t::feature::TABLETS)) {
         fcfg._disabled_features.insert("TABLETS"s);
     }
+    if (!cfg.uuid_sstable_identifiers_enabled()) {
+        fcfg._disabled_features.insert("UUID_SSTABLE_IDENTIFIERS"s);
+    }
```

Comment:
There's another way to look at is: the cluster features says whether nodes support reading and writing sstables, and the db config feature says whether this node will use uuid or not. With this way of looking at things, db::config should not influence the cluster feature.

The main advantage is that it's easier to make it LiveUpdate. I guess it also makes sense in that it's impossible to disable cluster features after the fact.

Wait, maybe this patch is broken. If a cluster is started with uuid ssstable identifiers, then restarted without it, won't the nodes be rejected since they support an accepted cluster feature?